tests/test_admin.py

```diff
@@ -0,0 +1,53 @@
+from cms.test_utils.testcases import CMSTestCase
+
+from djangocms_alias.models import Alias as AliasModel, AliasContent, Category
+from djangocms_alias.utils import is_versioning_enabled
+
+
+class FiltersTestCase(CMSTestCase):
+    def setUp(self):
+        self.superuser = self.get_superuser()
+
+    def test_language_filter(self):
+        category = Category.objects.create(name='Language Filter Category')
+        alias = AliasModel.objects.create(
+            category=category,
+            position=0,
+        )
+        expected_en_content = AliasContent.objects.create(
+            alias=alias,
+            name="EN Alias Content",
+            language="en",
+        )
+        expected_de_content = AliasContent.objects.create(
+            alias=alias,
+            name="DE Alias Content",
+            language="de",
+        )
+        # If versioning is enabled be sure to create a version
+        if is_versioning_enabled():
+            from djangocms_versioning.models import Version
+
+            Version.objects.create(content=expected_en_content, created_by=self.superuser)
```

Comment:
There are versioning factories available in djangocms_versioning.test_utils.factories

Reply:
Unfortunately, they can't be used how you would expect, we will need to do some rearranging to make them reusable as they require the same test apps to be installed as versioning has.

Reply:
Ahh yeah of course, will approve now.